io/net/virt-net/network_virtualization.py

```diff
@@ -114,6 +114,10 @@ def setUp(self):
             self.cancel('Backing Device counts and vios name counts differ')
         self.backingdev_count = len(self.backing_adapter)
         self.bandwidth = self.params.get("bandwidth", '*', default=None)
+        self.vnic_priority = self.params.get("priority", '*', default=50).split(',')
```

Comment:
Do not assign a default here.
If it is left empty, create a default list full of 50s.
If not empty, test the length, and use the values.
Makes sense ?

Reply:
perfect, 
if empty : will create a list of 50s with x elements, where x=number of sriov_adapters
if notempty: I am already checking the length here 
        if len(self.vnic_priority) != len(self.backing_adapter):
            self.cancel('Backing Device counts and priority counts differ')

Reply:
perfect